docs/src/InvariantTheory/finite_groups.md

```diff
@@ -89,16 +89,18 @@ julia> M2 = matrix(K, [1 0 0; 0 a 0; 0 0 -a-1])
 [0   0   -a - 1]
 
 julia> G = matrix_group(M1, M2)
-Matrix group of degree 3 over K
+Matrix group of degree 3
+  over cyclotomic field of order 3
 
 julia> IR = invariant_ring(G)
 Invariant ring of
-Matrix group of degree 3 over K
+  Matrix group of degree 3 over cyclotomic field of order 3
 with generators
-AbstractAlgebra.Generic.MatSpaceElem{nf_elem}[[0 0 1; 1 0 0; 0 1 0], [1 0 0; 0 a 0; 0 0 -a-1]]
+  AbstractAlgebra.Generic.MatSpaceElem{nf_elem}[[0 0 1; 1 0 0; 0 1 0], [1 0 0; 0 a 0; 0 0 -a-1]]
```

Comment:
maybe this could be printed properly as matrix...

Reply:
I do not think that this stays readable if we print it as a full-blown matrix and there are multiple generators.
Removing the type would be great

Reply:
I don't see a problem with printing proper matrices (neatly one over the other like Magma does). Sure, this will quickly fill the screen (just type `ShephardTodd(37);` in Magma), but if the user (that would be me) asks for how the group `G` in the REPL was defined again, then they should get a readable answer. I personally don't consider the "folded up" way to print a matrix in one line readable. We could of course put `⋮` in a longer list of generators, like it was proposed in the HackMD (is implemented?) for ideals in polynomial rings.

Reply:
Okay, I just noticed that this is about the printing of `InvRing`. `MatrixGroup` still does not print any generators.

Reply:
But I was thinking about MatrixGroup